wagtailio/static/sass/base/_typography.scss

```diff
@@ -76,6 +76,27 @@ h4,
     }
 }
 
+.teaser-heading {
+    font-size: 1.25rem;
+    line-height: 1.5rem;
+    font-weight: $weight--bold;
+
+    @include media-query(large) {
+        font-size: 1.875rem;
+        line-height: 2.438rem;
```

Comment:
In the Figma file i'm seeing a font size of 30px  and a line height of 39px for the heading:

![Screenshot 2022-07-27 at 14 03 00](https://user-images.githubusercontent.com/31627284/181255232-82f8447e-fc45-443a-b191-434e2a079ce9.png)